CMakeLists.txt

```diff
@@ -122,7 +122,7 @@ endif()
 # Require C++17's std::filesystem
 # This is fully standard, but links a separate library when using old compilers
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
-find_package(Filesystem REQUIRED)
+find_package(Filesystem REQUIRED COMPONENTS Experimental Final)
```

Comment:
I think it'll live forever in old gcc and clang version, but eventually not being used any more once all software stacks are more up to date to include the final one.
In any case, this logic should be fine because `final` will take precedence over `experimental` if both are found.

Reply:
Yep, if this works with current compilers it should work as-is indefinitely.